contrib/forms/evaluation/FormEvaluation.class.php

```diff
@@ -286,12 +286,12 @@ function persist()
 
         parent::persist();
         if (is_numeric($this->id) and !empty($this->checks)) {
-            $sql = "delete FROM form_evaluation_checks where foreign_id = '" . $this->id . "'";
-            sqlQuery($sql);
+            $sql = "delete FROM form_evaluation_checks WHERE foreign_id = ?";
+            sqlQuery($sql, array($this->id));
             foreach ($this->checks as $check) {
                 if (!empty($check)) {
-                    $sql = "INSERT INTO form_evaluation_checks set foreign_id='"  . add_escape_custom($this->id) . "', name = '" . add_escape_custom($check) . "'";
-                    sqlQuery($sql);
+                    $sql = "INSERT INTO form_evaluation_checks set foreign_id= ?, name = ?";
+                    sqlQuery($sql, array(add_escape_custom($this->id), add_escape_custom($check)));
```

Comment:
Remove the add_escape_custom() wrapper.
(note this is a function that will sql escape variables; but binding is better)